Assets/Item.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Item : MonoBehaviour {
+
+	public float attack_increase;
+	public bool ice_fire;
+	public int increase_fireball_number;
+	// Use this for initialization
+	void OnTriggerEnter(Collider other){
```

Comment:
And if two guy get into this function at the same time.
This may have two effect.
Use a lock somehow.